packages/react/types/carbon.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'carbon-components' {
```

Comment:
Should we also add types to these packages?

Reply:
Good question, I think it's probably best to figure out the process in the package where it makes the biggest difference then add it where helpful.